security/pkg/stsservice/tokenmanager/google/mock/mockserver.go

```diff
@@ -58,10 +56,10 @@ type federatedTokenResponse struct {
 }
 
 type accessTokenRequest struct {
-	Name      string            `json:"name"`
-	Delegates []string          `json:"delegates"` // nolint: structcheck, unused
-	Scope     []string          `json:"scope"`
-	LifeTime  duration.Duration `json:"lifetime"` // nolint: structcheck, unused
+	Name      string        `json:"name"`
+	Delegates []string      `json:"delegates"` // nolint: structcheck, unused
+	Scope     []string      `json:"scope"`
+	LifeTime  time.Duration `json:"lifetime"` // nolint: structcheck, unused
```

Comment:
is duration.Duration something we have to change as part of upgrade?

Reply:
wireformat should't matter right?

Reply:
Yes because the usage is wrong (if I understand it right). This isn't protobuf - why is it using a protobuf duration? I am pretty sure the original dev just autocompleted the wrong type

Reply:
Also note this is literally not used (`nolint: structcheck, unused`) which is why it wasn't noticed to be broken before